src/details-element-polyfill/polyfill.coffee.erb

```diff
@@ -71,22 +71,21 @@ polyfillToggle = ->
       summary.setAttribute("aria-expanded", true)
 
 polyfillFocusAndARIA = ->
-  for summary in document.querySelectorAll("summary")
+  for summary in findElementsWithTagName(document, "SUMMARY")
```

Comment:
I think we need to pass `document.documentElement` here or update `findElementsWithTagName()` because `document.nodeType` != `Node.ELEMENT_NODE`.